common/src/test/java/io/netty/util/RecyclerTest.java

```diff
@@ -24,7 +24,7 @@
 
 public class RecyclerTest {
 
-    @Test(expected = IllegalStateException.class)
+    //@Test(expected = IllegalStateException.class)
```

Comment:
but the same way user may not be correct and operate on dead buffers without ensureAccessible...so maybe a setting as usual